code-challenges.js

```diff
@@ -12,26 +12,65 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+const divide = (num) => {
+  if (num % 3 === 0) {
+    return `"${num} is divisible by three"`
+// I had single qoutes and not back ticks..... ***
+  }else{
+    return `"${num} is not divisible by three"`
+  }
+}
+console.log(divide(num1));
+console.log(divide(num2));
+console.log(divide(num3));
 
 
 
-
-// --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
-
-
+// need a const that returns every variable capitalized
+const capital = (array) => {
+  let array42 = array.map(value => {
```

Comment:
As Rachael mentioned during review: Lets make sure our variables we are declaring are descriptive and communicate clearly what they are for. Right now, at first glance, I have no idea what array42 means.